CantoboardFramework/Keyboard/View/KeyCap.swift

```diff
@@ -327,20 +327,20 @@ indirect enum KeyCap: Equatable, ExpressibleByStringLiteral {
         case "9": return ["九", "9", "玖", "９", "⑨", "⑼", "⒐", "❾", "㊈", "㈨"]
         case "0": return ["0", "０", "零", "十", "拾", "⓪", "°"]
         // 123 2nd row
-        case "-": return ["-", "－", "–", "—", "•"]
```

Comment:
Could you keep this as to match iOS English keyboard?

Reply:
OK.